### DISCUSSION TITLE
能否让页面在切换时返回顶部？

### DISCUSSION THREAD

**Discussion by zwying0814** [PaperStrike/Pjax#367] [Q&A] [answered]
目前如果设置 `scrollTo:false`，页面切换时会停留在原地。
设置 `scrollTo:true`能回到页面顶部，但会有滚动，这往往与某些lazyload场景冲突。
能否直接返回页面顶部，而没有滚动到页面的效果？

**Comment by PaperStrike** [comment, marked as answer]
Pjax 目前不会操控滚动过渡效果，更可能是你页面 CSS 误指定了 [`scroll-behavior`](https://developer.mozilla.org/en-US/docs/Web/CSS/scroll-behavior)。
